src/App.js

```diff
@@ -2,44 +2,59 @@ import React from "react";
 import ReactDOM from "react-dom/client";
 import logo from "../images/The-Incredible-Hulk-2-icon.png"
 import userIcon from "../images/Avengers-Hulk-icon.png"
+import data from "./data.json"
+import CardComponent from "./CardComponent";
 
-/**
- * Header with nested react create elememt without using JSX
- *
- * const reactHeading = React.createElement("div", { className: "title p-3" }, [
- * React.createElement("h2", {}, "Namaste React Bootcamp"),
- * React.createElement("h3", {}, "HUSTLERS")]);
- */
 
-/**
- * Header by using react functional component
- */
-
-const TitleComponent = () => {
-  return <h1 className="title">HUSTLERS</h1>;
-};
+/** header by using react functional component */
+const title = "HUSTLERS";
 
 const SearchBar = () =>{
     return(
         <form className="search-bar">
-            <input className="input-search" type="search" placeholder="Search the Hustler.." />
+            <input className="input-search" type="search" placeholder="Search the Hustler..." />
             <button type="submit"><i className="fa fa-search fa-2x"></i></button>
         </form>
     )
 };
 
-/** Header */
+/** header component*/
 const HeaderComponent = () => {
     return(
         <div className="header">
          <img src={logo}/>
-         <TitleComponent/>
+         <h1 className="title">{title}</h1>
          <SearchBar/>
          <img className="user-icon" src={userIcon}/>
         </div>
-    )
+    );
 };
 
-/** Accessing DOM to render react element*/
+const CardContainer = () => {
+    const cards = data.map((member) =>{
+        return  <CardComponent members={member}/>
+    })
+   return cards;
+}
+
+/** body component */
+const BodyComponent = ()=>{
```

Comment:
If making it a component not a container, better to make it like -
``` 
const BodyComponent = ({children}) => {
    return (
    <div class="cardContainer">
    {children}
    </div>
    )
}
```

Then wrap where you want to render this component as 
```
<BodyComponent><CardContainer></BodyComponent>
```